share/completions/smerge.fish

```diff
@@ -0,0 +1,13 @@
+# Sublime Merge CLI tool
+
+complete -c smerge -s n -l new-window -d 'Open a new window'
+complete -c smerge -l launch-or-new-window -d 'Open new window only if app is running'
+complete -c smerge -s b -l background -d "Don't activate the application"
+complete -c smerge -l safe-mode -d 'Launch in a sandboxed environment'
+complete -c smerge -s h -l help -d 'Show help (this message) and exit'
+complete -c smerge -s v -l version -d 'Show version and exit'
+complete -c smerge -a search -x -d 'Search for commits in the current repository'
```

Comment:
What about suggesting commits like [here](https://github.com/fish-shell/fish-shell/blob/master/share/completions/git.fish#L30)?

Reply:
I don't think that would be useful. The expected argument to `search` is a query string; the subcommand launches Sublime Merge with its argument pre-filled into the commit _message_ search field, which doesn't search commit hashes.